var/spack/repos/builtin/packages/python/package.py

```diff
@@ -267,7 +268,6 @@ class Python(Package):
     )
     conflicts('+tix', when='~tkinter',
               msg='python+tix requires python+tix+tkinter')
-
```

Comment:
Can you restore this line?

Reply:
Other places I see no space between two conflicts statemetns, so I removed the existing spaces 

--snippet--
    # CPython tries to build an Objective-C file with GCC's C frontend
    # https://github.com/spack/spack/pull/16222
    # https://github.com/python/cpython/pull/13306
    conflicts('%gcc platform=darwin',
              msg='CPython does not compile with GCC on macOS yet, use clang. '
                  'See: https://github.com/python/cpython/pull/13306')
    # For more information refer to this bug report:
    # https://bugs.python.org/issue29712
    conflicts(
        '@:2.8 +shared',
        when='+optimizations',
        msg='+optimizations is incompatible with +shared in python@2.X'
    )
    conflicts('+tix', when='~tkinter',
              msg='python+tix requires python+tix+tkinter')
    conflicts('%nvhpc')
    conflicts('@:2.7', when='platform=darwin target=aarch64:',
              msg='Python 2.7 is too old for Apple Silicon')

--snippet--

Let me know if you think otherwise.